lib/util.js

```diff
@@ -371,10 +371,11 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
+  var symbolKeys = Object.getOwnPropertySymbols(value);
```

Comment:
I haven’t run this but right now this looks like this will include even *non-*enumerable Symbol properties?